styles/thumbnail.css

```
@@ -1,14 +1,17 @@
 @import './colors.css';
 
 .thumbnail-list {
-  margin: 0 auto 56px;
   display: grid;
-  grid-template-columns: repeat(5, 200px);
-  gap: 70px;
+  justify-content: center;
+
+  width: 100%;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)